guides/common/modules/proc_running-an-ansible-playbook.adoc

```diff
@@ -0,0 +1,27 @@
+[id="running-an-ansible-playbook-from-{project-context}_{context}"]
+= Running an Ansible playbook from {Project}
+
+You can run an Ansible playbook on a host or host group by executing a remote job in {Project}.
+
+.Prerequisites
+* Ansible plugin in {Project} is enabled.
+* You have configured xref:Configuring_and_Setting_Up_Remote_Jobs_{context}[remote job execution].
+* You have an Ansible playbook ready to use.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Monitor* > *Jobs*.
+. Click *Run Job*.
+. In *Job category*, select `Ansible Playbook`.
+. In *Job template*, select `Ansible - Run playbook`.
+. Click *Next*.
+. Select the hosts on which you want to run the playbook.
+. In the *playbook* field, paste the content of your Ansible playbook.
```

Comment:
your wording is spot on; however, just naming the field "playbook" in the WebUI is a bit "hmmm".